JASP-Engine/JASP/R/mlclassificationknn.R

```diff
@@ -78,9 +78,12 @@ MLClassificationKNN <- function(dataset=NULL, state = NULL, options, perform="ru
         
     }
     
-    # set the seed so that every time the same set is chosen (to prevent random results) ##
-    
-    set.seed(options[["seed"]])
+    # set the seed ##
+    if(options[["seedBox"]]){
+        set.seed(options[["seed"]])
+    } else {
+        set.seed(Sys.time())
```

Comment:
Hi @koenderks can you add the **seed** to state? (In the future, we will be able to use the seed for unit tests)